var/spack/repos/builtin/packages/py-slurm-pipeline/package.py

```diff
@@ -0,0 +1,42 @@
+##############################################################################
+# Copyright (c) 2013-2018, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/spack/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class PySlurmPipeline(PythonPackage):
+    """A Python class for scheduling SLURM jobs"""
+
+    homepage = "https://github.com/acorg/slurm-pipeline"
+    url      = "https://pypi.io/packages/source/s/slurm-pipeline/slurm-pipeline-1.1.13.tar.gz"
+
+    version('1.1.13', 'd1f8c78a64718ec5e2e40ba1b6816017')
+
+    depends_on('py-setuptools', type='build')
+    # using open ranges although requirements*.txt give explicit versions
+    depends_on('py-discover@0.4.0:',   type=('build', 'run'))
+    depends_on('py-mock@1.3.0:',   type=('build', 'run'))
```

Comment:
`mock` is only required for unit tests, not actually required to build or run the package. You can remove this.